code/game/objects/items/airbubble.dm

```diff
@@ -357,6 +351,9 @@
 			return
 		user.drop_item()
 	else if(istype(W, /obj/item/weapon/handcuffs/cable))
+		if(zipped)
+			user.visible_message("<span class='warning'>[src]'s zipper is already restrained.</span>")
```

Comment:
This should be `to_chat`, not vis_message.

Reply:
This is only visible to user, as it should be.

Reply:
No, it's visible to everyone around him. The first arg of `visible_message` is broadcast to everyone in range. If no second arg is given, it's also sent to the user. You'd either have to do `user.visible_message("", "this blob")` or `to_chat(user, "blob")`. You should do the latter because it's clearer code that way.